BackEnd/tests/DeleteProjectTest.php

```diff
@@ -0,0 +1,204 @@
+<?php
+
+namespace HostMyDocs\Tests;
+
+class DeleteProjectTest extends BaseTestCase
+{
+    /**
+     * @var array credentials to be inserted into environment
+     */
+    private $serverCredentials = 'beep:bep';
+
+    private $wrongCredentials = 'plop:plop';
+
+    /**
+     * @dataProvider arrayOfParametersProvider
+     */
+    public function testDeleteProject($projectsToPost, $parameters, $credentialsArray, $statusCode)
+    {
+        foreach ($projectsToPost as $project) {
+            $files = ['archive' => $this->createZipFile()];
+            $credentials= $this->serverCredentials;
+            $this->runApp('POST', '/addProject', $parameters, $files, $credentials);
+        }
+
+        $credentials = null;
+
+        if (array_key_exists('serverCredentials', $credentialsArray)) {
+            putenv('CREDENTIALS='.$credentialsArray['serverCredentials']);
+        }
+
+        if (array_key_exists('userCredentials', $credentialsArray)) {
+            $credentials = $credentialsArray['userCredentials'];
+        }
+
+        $response = $this->runApp('DELETE', '/deleteProject', $parameters, null, $credentials);
+        $this->assertEquals($statusCode, $response->getStatusCode());
+    }
+
+    /**
+     * Providing parameters for requests
+     */
+    public function arrayOfParametersProvider()
```

Comment:
Maybe you could add some cases to increase test coverage ?

(each line starting with a red squared is not ran while tests)

![image](https://user-images.githubusercontent.com/6715182/40472805-c84d596c-5f3a-11e8-9a0c-46b30c9a4f30.png)

Reply:
last should cover at least these lines